packages/ui/package.json

```diff
@@ -18,7 +18,8 @@
     "typecheck": "tsc --noEmit",
     "size": "size-limit",
     "test:unit": "LC_ALL=en_US.UTF-8  pnpm vitest --run --config vite.config.ts",
-    "test:unit:coverage": "pnpm test:unit --coverage"
+    "test:unit:coverage": "pnpm test:unit --coverage",
+    "publint": "npx publint"
```

Comment:
si on l'utilise dans la ci + husky, tu peux l'installer dans le package root

Reply:
J'ai mis dans le package root + sub package dans le cas ou tu veux le run localement, et pas bete pour husky je vais l'ajouter

Reply:
actually not the best idea, publint needs a build in order to check exports matches the build folder. If I was to add it into husky before commit it will need to build all packages locally